### PR TITLE
Updating references, ORCID and affiliation.

### PR DESCRIPTION
**Pull request by arfon**
Hi @alk224, these edits are required to make your submission JOSS-compatible :grin:

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -6,17 +6,16 @@ tags:
   - amplicon sequencing
 authors:
  - name: Andrew Krohn
-   orcid: 0000-0002-3957-2474
-   affiliation: GitHub Inc.
+   orcid: 0000-0002-7057-9957
```
Comment (arfon) [inline paper.md:9]: Updating with your ORCID

File: paper.md [hunk 2]
```diff
@@ -6,17 +6,16 @@ tags:
...
+   affiliation: Northern Arizona University
```
Comment (arfon) [inline paper.md:10]: Also, don't think you want your affiliation as 'GitHub' here.

File: paper.md [hunk 3]
```diff
@@ -6,17 +6,16 @@ tags:
...
...the sequenced region) and provides new functions for fastq/a file and OTU table manipulations (*e.g.*, fastq file concatenation, fastq/a length histograms, filtering fastq/a by length, filtering OTU tables by observation). It automates tasks commonly associated with data analysis through QIIME as long as certain conventions are met. The main functions are pick_otus (takes raw fastq data all the way to an OTU table), align_and_tree (alignment of representative sequences and phylogenetic tree construction), and core_diversity (production of graphs and statistical analyses). The core_diversity output provides access to all input files and transformed derivatives, a well-organized output including sequences extracted for each OTU, phylum-level phylogenetic tree representation via phyloseq [@mcmurdie2013phyloseq], and analyses performed with rarefied data, or using the popular cumulative sum scaling [@paulson2013differential] or DESeq2 [@love2014moderated] data normalizations. In addition to standard QIIME output, statistical tests of differential abundance are provided via indicator species analysis [@de2009associations] and analysis of composition of microbiomes [@mandal2015analysis].
```
Comment (arfon) [inline paper.md:17]: All I've done here is make these proper Markdown references (i.e. link to the entries in the bibTeX).

**Comment by alk224** [comment]
Thanks you so much for the help. I merged the pull request and made one more change to the affiliation.

**Pull request merged**